### PR TITLE
Update to @clearlydefined/spdx v0.1.8

### PR DESCRIPTION
**Pull request by qtomlinson**
- Update tests for v0.1.8, to reflect https://github.com/clearlydefined/spdx/pull/30
- Also added more tests
- This is a interim step before moving to use GitHub packages.

### CODE REVIEW

**Comment by qtomlinson** [comment]
@ljones140 I [experimented](https://github.com/qtomlinson/service/commit/616a1352670b3435fb3284551dfb28c1aeacdfb7) with using GitHub packages and found that an environment variable GITHUB_TOKEN may be required for authentication to install GitHub packages.  See [documentation](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry).  Referring to GitHub repo with tag for dependency is an interim step to consume the changes.

**Comment by ljones140** [comment]
> @ljones140 I [experimented](https://github.com/qtomlinson/service/commit/616a1352670b3435fb3284551dfb28c1aeacdfb7) with using GitHub packages and found that an environment variable GITHUB_TOKEN may be required for authentication to install GitHub packages. See [documentation](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry). Referring to GitHub repo with tag for dependency is an interim step to consume the changes.

Hey @qtomlinson 

I don't think we will need a `GITHUB_TOKEN` as this package is public, so we don't need authn to read.

We do need to tell NPM to use GitHub as the package repository for `@clearlydefined` via the `.npmrc`.
`@clearlydefined:registry=https://npm.pkg.github.com`

I've branched off this and made a commit here which does this https://github.com/clearlydefined/service/commit/43092e915876a708b72de6942170e67adeac9109

File: test/lib/spdx.js [hunk 1]
```diff
@@ -217,13 +217,17 @@ describe('SPDX utility functions', () => {
       'NOASSERTION': 'NOASSERTION',
       'See license': 'NOASSERTION',
       'MIT OR Apache-2.0': 'MIT OR Apache-2.0',
+      'MIT or Apache-2.0': 'NOASSERTION',
```
Comment (elrayle) [inline test/lib/spdx.js:220]: Is this what we expect or want?  Shouldn't operators be case insensitive?

**Review by elrayle — APPROVED** [review]
There is one new test that I am surprised maps to NOASSERTION simply because of the case of the operator.  This is not a blocker for this PR since it is only showing how the code actually works now.  But I'd be inclined to open an issue requesting case-insensitivity for operators.

File: test/lib/spdx.js (hunk 1 above)
Comment (qtomlinson) [reply test/lib/spdx.js:220]: According to SPDX specification, operator has to be upper cased, whereas, the license identifiers are case insensitive.  So our code is behaving correctly.  I added the case here as documentation and for my future fix for curation validation.

**Pull request merged**